lib/Echidna/Config.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DeriveGeneric, TemplateHaskell #-}
+
+module Echidna.Config where
+
+import Control.Lens
+import Data.Aeson
+--import Data.DoubleWord (Word256)
+import GHC.Generics
+
+import qualified Data.ByteString.Char8 as BS
+import qualified Data.Yaml as Y
+
+import EVM.Types (W256)
+
+
+data Config = Config
+  { _solcArgs :: Maybe String
+  , _epochs :: Int
+  , _testLimit :: Int
+  , _range :: Int
+  , _gasLimit :: W256 }
+  deriving (Show, Generic)
+
+makeLenses ''Config
+
+instance FromJSON Config
+
+defaultConfig :: Config
+defaultConfig = Config
```

Comment:
Do we document what these fields mean anywhere? I'm not sure the names would make that obvious to a new user